<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>370</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>370</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>370</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="Temperature">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>346</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="ALL">
     <item>
      <layout class="QVBoxLayout" name="Title">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Bahnschrift SemiLight SemiConde</family>
           <pointsize>27</pointsize>
           <italic>true</italic>
           <bold>true</bold>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="text">
          <string>Raspberry  Pi  Temperature  Monitor</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="Temp">
       <property name="topMargin">
        <number>50</number>
       </property>
       <property name="rightMargin">
        <number>60</number>
       </property>
       <property name="bottomMargin">
        <number>50</number>
       </property>
       <item>
        <widget class="QLabel" name="TEMP">
         <property name="font">
          <font>
           <family>Bahnschrift SemiLight SemiConde</family>
           <pointsize>70</pointsize>
           <bold>false</bold>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="text">
          <string>TEMP:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber">
         <property name="palette">
          <palette>
           <active>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>212</red>
               <green>63</green>
               <blue>63</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>212</red>
               <green>63</green>
               <blue>63</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled/>
          </palette>
         </property>
         <property name="font">
          <font>
           <family>Microsoft New Tai Lue</family>
           <pointsize>12</pointsize>
           <bold>false</bold>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="digitCount">
          <number>6</number>
         </property>
         <property name="segmentStyle">
          <enum>QLCDNumber::Flat</enum>
         </property>
         <property name="value" stdset="0">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="UserInformation">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="IP_2">
         <item>
          <widget class="QLabel" name="IP">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Microsoft New Tai Lue</family>
             <pointsize>12</pointsize>
             <bold>false</bold>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="text">
            <string>IP:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="IPEdit">
           <property name="palette">
            <palette>
             <active>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>59</red>
                 <green>59</green>
                 <blue>59</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>59</red>
                 <green>59</green>
                 <blue>59</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled/>
            </palette>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="User_2">
         <item>
          <widget class="QLabel" name="User">
           <property name="font">
            <font>
             <family>Microsoft New Tai Lue</family>
             <pointsize>12</pointsize>
             <bold>false</bold>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="text">
            <string>User:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="UserEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="Password_2">
         <item>
          <widget class="QLabel" name="Password">
           <property name="font">
            <font>
             <family>Microsoft New Tai Lue</family>
             <pointsize>12</pointsize>
             <bold>false</bold>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="text">
            <string>Password:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="PasswordEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="Confirm">
       <property name="leftMargin">
        <number>50</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>50</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QPushButton" name="START">
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="palette">
          <palette>
           <active>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>25</red>
               <green>25</green>
               <blue>25</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>25</red>
               <green>25</green>
               <blue>25</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled/>
          </palette>
         </property>
         <property name="font">
          <font>
           <family>MS Sans Serif</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="text">
          <string>START</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="STOP">
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="palette">
          <palette>
           <active>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>25</red>
               <green>25</green>
               <blue>25</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>25</red>
               <green>25</green>
               <blue>25</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled/>
          </palette>
         </property>
         <property name="font">
          <font>
           <family>MS Sans Serif</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="text">
          <string>STOP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="EXIT">
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="palette">
          <palette>
           <active>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>25</red>
               <green>25</green>
               <blue>25</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>25</red>
               <green>25</green>
               <blue>25</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled/>
          </palette>
         </property>
         <property name="font">
          <font>
           <family>MS Sans Serif</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="text">
          <string>EXIT</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
